tests/integration/create_product.t

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/perl -w
+
+use ProductOpener::PerlStandards;
+
+use Test::More;
+use ProductOpener::APITest qw/:all/;
+use ProductOpener::Test qw/:all/;
+
+remove_all_products();
+wait_dynamic_front();
+
+my $admin_ua = new_client();
+my $anon_ua = new_client();
+
+my %product_fields = (
+	code => '200000000099',
+	lang => "en",
+	product_name => "Testttt-75ml",
+	generic_name => "Tester",
+	quantity => "75 ml",
+	link => "https://github.com/openfoodfacts/openfoodfacts-server",
+	expiration_date => "test",
+	ingredients_text => "apple, milk",
+	origin => "france",
+	serving_size => "10g",
+	packaging_text => "no",
+	action => "process",
```

Comment:
action and type fields can be added by the create_product() function as we always need them.

Reply:
in fact as @alexgarel suggested, we can use the %default_product_form approach to add action and type, so you can disregard this comment